### DISCUSSION TITLE
External symbol not resolved error. Microsoft Visual Code 2019.

### DISCUSSION THREAD

**Discussion by marius-sucan** [nothings/stb#1579] [Q&A] [answered]
Hello!

I added this line in my .cpp:
#include "stb_image_resize2.h"

And:
unsigned char *r = stbir_resize_uint8_linear(bitmapData, w, h, oStride, otherData, w2, h2, nStride, STBIR_BGRA);

I get an error now when I compile:
External symbol not resolved. LNK 2001 and LNK 1120.

Any ideas why... and how to fix this?

Thank you .

Best regards, Marius.

**Comment by nothings** [comment, marked as answer]
```
#define STB_IMAGE_RESIZE_IMPLEMENTATION
#include "stb_image_resize2.h"
```

**Reply by marius-sucan** [reply]
Thank you . It compiled fine!
